authors/apps/articles/serializers.py

```diff
@@ -28,10 +52,46 @@ def validate_user_permissions(self, request, data):
                 status=status.HTTP_403_FORBIDDEN
             )
 
+    def validate_tag_list(self, validated_data):
```

Comment:
Good work validating that the input is a List, I would request you kindly validate that the list items are strings